docs/stores/eks/eks.md

```diff
@@ -0,0 +1,51 @@
+# EKS store
```

Comment:
For this to work, the AWS CLI itself needs to be set up with credentials for the right account and region right? 
That's what I locally do with  `aws configure`. Could you add a sentence or two about that? 
Thanks!

Reply:
Also, how does the AWS SDK get the credentials to create the client used to fetch the EKS clusters? Are there some sort of default credentials on the filesystem created by the AWS CLI?

Reply:
The sdk uses the default chain to get credentials (instance-metadata, environment, configuration file), but in this case here to support multiple stores it's made to work only with named profiles (the `~/.aws/credentials` file).

Reply:
Okay, good to know. Could you add that info to the documentation? Thanks!